decred/tests/unit/dcr/test_dcrdata_unit.py

```diff
@@ -128,11 +128,15 @@ def test_checkoutput():
         checkOutput(tx, 0)
 
 
-class MockWebsocketClient:
+class MockWebSocketClient:
```

Comment:
Changing the capitalization to be consistent with the `websocket.WebSocketApp` class that we're using, and also because `WebSockets` is the way it is commonly stylized.